test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ObjectExplorerServiceTests.cs

```diff
@@ -78,7 +78,13 @@ public ObjectExplorerServiceTests()
         public async Task CreateSessionRequestReturnsSuccessAndNodeInfo()
```

Comment:
Already added it in integration test

---
In reply to: [112104491](https://github.com/Microsoft/sqltoolsservice/pull/322#discussion_r112104491) [](ancestors = 112104491)